src/main/java/teammates/ui/webapi/CreateFeedbackSessionAction.java

```diff
@@ -62,7 +62,11 @@ public JsonResult execute() {
 
         try {
             logic.createFeedbackSession(fs);
-        } catch (EntityAlreadyExistsException | InvalidParametersException e) {
+        } catch (EntityAlreadyExistsException e) {
+            throw new InvalidHttpRequestBodyException("A session named " + feedbackSessionName
```

Comment:
This is not exactly `InvalidHttpRequestBodyException`, because the body _is_ valid, but an external factor (in this case the state of DB) causes the request to be invalid. However, we currently lack the exception type to encapsulate this kind of Exception, so this can stay for now.

Reply:
#11368 has introduced a new exception wrapper (in fact also changing this line), so you'll soon find that you only need to include the custom error message and not separating the `catch` block anymore